sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -97,6 +97,13 @@ public partial class InteractiveBrowserCredential : Azure.Core.TokenCredential
         public override Azure.Core.AccessToken GetToken(Azure.Core.TokenRequestContext requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public override System.Threading.Tasks.ValueTask<Azure.Core.AccessToken> GetTokenAsync(Azure.Core.TokenRequestContext requestContext, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public static partial class KnownAuthorityHosts
```

Comment:
On the name, python has KnownAuthorities and .NET has KnownAuthorityHosts.  I like the .NET version better, but should it be consistent with other langs?

Reply:
I intentionally left this different, as the inconsistency already exists. The option names are different. In python the kwarg which configures this is `authority`
~~~ python
credential = DefaultAzureCredential(authority=KnownAuthorities.AZURE_CHINA)
~~~
whereas in .NET it's `AuthorityHost`
~~~ c#
var options = new DefaultAzureCredentialOptions { AuthorityHost = KnownAuthorityHosts.AzureChinaCloud };

var client = new KeyClient(new Uri(keyVaultUrl), new DefaultAzureCredential(options));
~~~

Given that we can't change the property name on the options, I think it's more important that the class match this than the class name in a different language.

Reply:
Sounds good.  Thanks